main.js

```diff
@@ -0,0 +1,48 @@
+const books = [];
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const button = document.querySelector('.btn');
+
+// REMOVE BOOK FUNCTION
+function deleteBook(el) {
+  if (el.classList.contains('remove')) {
+    el.parentElement.parentElement.remove();
+  }
+}
```

Comment:
Your Function to delete a book object is working perfectly, but only removing the book from the DOM (disappear from your page).  👍🏼 

However, You need to delete the object from your `books` collection too. as stated here...

![e5ecde40-f0ce-4b53-bc05-feadab708ec9](https://user-images.githubusercontent.com/24830039/128772557-4a459250-8754-436f-bce1-5e9201991d3b.png)

for demonstration: please observe the book variable in the three situations

![Untitled](https://user-images.githubusercontent.com/24830039/128774335-ca4b7425-566e-4ec7-920b-848e20049a6c.png)